project/apps/matching-service/src/db/match.redis.ts

```diff
@@ -90,7 +94,7 @@ export class MatchRedis {
 
   /**
    * Fetches a match request from redis
-   * @param match_req_id The matchReqId to fetch
+   * @param match_req_id The match_req_id to fetch
    * @returns The match request if found, otherwise null
    */
   async getMatchRequest(match_req_Id: string): Promise<MatchRequestDto | null> {
```

Comment:
unrelated:
could you also help to remove this log in this PR

https://github.com/CS3219-AY2425S1/cs3219-ay2425s1-project-g30/blob/7d2c8cf13706d41d1c21dd852398962c5baa8b02/project/apps/matching-service/src/db/match.redis.ts#L105